src/storage/v2/CMakeLists.txt

```diff
@@ -43,4 +43,5 @@ add_library(mg-storage-v2 STATIC
         inmemory/replication/recovery.cpp
 )
 
+target_include_directories(mg-storage-v2 PUBLIC ${CMAKE_SOURCE_DIR}/include)
```

Comment:
What's that change for?

Reply:
I moved the search mode enum to `/include` in order to avoid two redundant definitions in `memgraph` and `mage` (see https://github.com/memgraph/memgraph/pull/1739#discussion_r1504182412). This line is needed to expose the enum to text_index.{cpp, hpp}.